src/framework/components/element/text-element.js

```diff
@@ -692,11 +692,21 @@ Object.assign(pc, function () {
                     }
 
                     if (data) {
+                        var kerning = 0;
+                        if (numCharsThisLine > 0) {
+                            var table = this._font._data.kerning;
```

Comment:
I think `this._font.data` is the public property

Reply:
The engine rtl and CanvasFont tests were actually failing with "TypeError: Cannot read property 'kerning' of undefined" on this line.

Using 'data.kerning' instead of '_data.kerning' solved the issue, but I can't really figure out why...